src/app/rpmostree-builtin-status.c

```diff
@@ -526,6 +543,9 @@ print_one_deployment (RPMOSTreeSysroot *sysroot_proxy,
     g_print ("%s", checksum);
   g_print ("\n");
 
+  if (custom_origin_description)
```

Comment:
How about `if (custom_origin_url && custom_origin_description)` here to be extra sure we only print this if we also printed a custom URL?

Reply:
I ended up adding some hardening in the code to canonicalize the empty string `""` to `NULL` in more places, and also fall back to the URL if there's no description set which should address your concern :arrow_down: